src/Integration/ProfileConflicts/RuleSetAggregate.cs

```diff
@@ -0,0 +1,55 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="RuleSetAggregate.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace SonarLint.VisualStudio.Integration.ProfileConflicts
+{
+    /// <summary>
+    /// Data-only class that represents that aggregated rule set information. 
+    /// Same rule sets should be represented by the same instance of <see cref="RuleSetAggregate"/> and have their <see cref="RuleSetDeclaration.ActivationContext"/>
+    /// associated with the shared instance by adding them into <see cref="ActivationContexts"/>.
+    /// </summary>
+    /// <seealso cref="RuleSetDeclaration"/>
+    public class RuleSetAggregate
```

Comment:
Confusing name: you've used "ruleSet" info for variables of this type in a couple of places, which seems like a better name.